polkadot/runtime/westend/src/weights/polkadot_runtime_parachains_inclusion.rs

```diff
@@ -17,25 +17,28 @@
 //! Autogenerated weights for `polkadot_runtime_parachains::inclusion`
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 32.0.0
-//! DATE: 2024-08-31, STEPS: `50`, REPEAT: `20`, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! DATE: 2025-02-12, STEPS: `50`, REPEAT: `20`, LOW RANGE: `[]`, HIGH RANGE: `[]`
 //! WORST CASE MAP SIZE: `1000000`
-//! HOSTNAME: `runner-svzsllib-project-674-concurrent-0`, CPU: `Intel(R) Xeon(R) CPU @ 2.60GHz`
-//! WASM-EXECUTION: `Compiled`, CHAIN: `Some("westend-dev")`, DB CACHE: 1024
+//! HOSTNAME: `73197ed99740`, CPU: `Intel(R) Xeon(R) CPU @ 2.60GHz`
+//! WASM-EXECUTION: `Compiled`, CHAIN: `None`, DB CACHE: 1024
 
 // Executed Command:
-// target/production/polkadot
+// frame-omni-bencher
+// v1
 // benchmark
 // pallet
-// --steps=50
-// --repeat=20
 // --extrinsic=*
+// --runtime=target/production/wbuild/westend-runtime/westend_runtime.wasm
+// --pallet=polkadot_runtime_parachains::inclusion
+// --header=/__w/polkadot-sdk/polkadot-sdk/polkadot/file_header.txt
+// --output=./polkadot/runtime/westend/src/weights
 // --wasm-execution=compiled
+// --steps=50
+// --repeat=20
 // --heap-pages=4096
-// --json-file=/builds/parity/mirrors/polkadot-sdk/.git/.artifacts/bench.json
-// --pallet=polkadot_runtime_parachains::inclusion
-// --chain=westend-dev
-// --header=./polkadot/file_header.txt
-// --output=./polkadot/runtime/westend/src/weights/
+// --no-storage-info
+// --no-min-squares
+// --no-median-slopes
```

Comment:
is there a rationale for these params? were they present before?

Reply:
It suppresses some analysis that is only printed to the CLI output. Does not affect weight files.